docs/version0.15.md

```diff
@@ -63,6 +64,10 @@ The ChaCha20 and ChaCha20-Poly1305 algorithms can now use OpenSSL on Java 11. Fo
 
 For compatibility, the [`-XX:OnOutOfMemoryError`](xxonoutofmemoryerror.md) OpenJDK Hotspot option is now supported by OpenJ9.
 
+### Support for automatic huge pages allocation
+
+The VM now supports automatic allocation of huge pages on Linux when you use the `madvise` setting. To enable this feature, set [`-XX:+TransparentHugePage`](xxtransparenthugepage.md) on the command line when you start your application. This option is currently not enabled by default.
```

Comment:
I think it will be better to mention what the madvise setting refer to (transparent hugepage option, as there is also a thp defrag that uses similar setting)

Reply:
`/sys/kernel/mm/transparent_hugepage/enabled`

Reply:
Added in brackets against `madvise`